indico_hub/server.py

```diff
@@ -205,3 +208,11 @@ def delete():
     db.session.commit()
     es.indices.delete(index='reg_data', ignore=[400, 404])
     return f'{num_rows}'
+
+
+@api.route('/api/crawl/<string:uuid>')
+def crawl(uuid):
```

Comment:
No need for a new crawl endpoint, just geolocate the IP when the instance gets registered.

Reply:
solved